hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OmMetadataGenerator.java

```diff
@@ -326,8 +339,18 @@ private void applyOperation(long counter) throws Exception {
     case CREATE_KEY:
       keyName = getPath(counter);
       getMetrics().timer(operation.name()).time(() -> {
-        try (OutputStream stream = bucket.createStreamKey(keyName,
-            dataSize.toBytes(), replicationConfig, new HashMap<>())) {
+        try (OutputStream stream = bucket.createKey(keyName,
+            dataSize.toBytes())) {
+          contentGenerator.write(stream);
+        }
+        return null;
+      });
```

Comment:
For example, this can be extracted to a separate function, e.g. `createKey(operation, bucket, keyName, dataSize)`

Reply:
Done,